defaults/main.yml

```diff
@@ -23,6 +23,11 @@ galaxy_tools_base_dir: /tmp
 # Blank variable to make sure it's defined
 galaxy_tools_api_key: ''
 
+# List of workflow files to be installed. Use empty list ([]) to skip installing workflows
+
+galaxy_tools_workflows:
+  - sample-workflow.ga
```

Comment:
How about switching this around and having the empty list be the default but including the example in the documentation line? 

Thanks for including the sample file.